ArjunaJTA/jta/classes/com/arjuna/ats/jta/logging/jtaLogger.java

```diff
@@ -33,47 +33,22 @@
 
 import org.jboss.logging.Logger;
 
-import java.lang.reflect.InvocationHandler;
-import java.lang.reflect.InvocationTargetException;
-import java.lang.reflect.Method;
-import java.lang.reflect.Proxy;
-
-public class jtaLogger implements InvocationHandler
+public class jtaLogger
 {
     public static final Logger logger = Logger.getLogger("com.arjuna.ats.jta");
-    public static final jtaI18NLogger i18NLogger = (jtaI18NLogger) Proxy.newProxyInstance(
-            jtaI18NLogger.class.getClassLoader(),
-            new Class[] { jtaI18NLogger.class },
-            new jtaLogger(Logger.getMessageLogger(jtaI18NLogger.class, "com.arjuna.ats.jta")));
+
+    /**
+     * For jtaI18NLogger, if any method, prefixed with <b>"warn_recovery"</b> is called then,
```

Comment:
I think this is a good comment but I consider it should be put into the class `jtaI18Logger` as it's the place where new methods for logging are added (and where the methods are named). I think when it's here it's better chance to be missed.

Reply:
Sure, I will move the comments to jtaI18Logger